cmflib/cmfquery.py

```diff
@@ -14,414 +14,577 @@
 # limitations under the License.
 ###
 
+import json
+import typing as t
+import logging
 import pandas as pd
 from ml_metadata.metadata_store import metadata_store
 from ml_metadata.proto import metadata_store_pb2 as mlpb
+
 from cmflib.mlmd_objects import CONTEXT_LIST
-import json
-import typing as t
 
+__all__ = ["CmfQuery"]
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: Rename to CMF Client or just set CmfClient = CmfQuery
```

Comment:
we have a command line interface to initialize cmf , push and pull metadata and artifacts , which we call client . So we cant call this as client .  And i dont think, this is a client , it just exposes the metadata stored.